semesterly/dev_credentials.py

```diff
@@ -10,41 +10,41 @@
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
 
-#DANGER: Should not be used in production
-#THE CREDENTIALS PROVIDED HERE ARE FOR TEST USE ONLY
-#THEY ARE NOT USED ON SEMESTER.LY NOR SHOULD THEY BE
+# DANGER: Should not be used in production
+# THE CREDENTIALS PROVIDED HERE ARE FOR TEST USE ONLY
+# THEY ARE NOT USED ON SEMESTER.LY NOR SHOULD THEY BE
 # USED IN ANY OTHER PRODUCTION ENVIRONMENT.
 
 
 SECRETS = {
-    #Credentials for a test application for Semester.ly (+ Google/Facebook)
-    'SECRET_KEY': 'ca&+%f9xv*$56z@8278xw*ci7_db9w=^dhkqmwu3as&30u%w+$',
-    'HASHING_SALT': "t9wg0p7x1$x5on89f*yk=tk*+$!g6g",
-    'GOOGLE_API_KEY': 'AIzaSyB4O0Z89wwSWdVFufZMmW7n05ftUaau9cs',
-    'SOCIAL_AUTH_GOOGLE_OAUTH2_KEY': '207925266161-qa90rek8apc0ablkjr3565um4u9kiht1.apps.googleusercontent.com',
-    'SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET': 'OiVIoVFTZrWXZ87tx7r1KJoo',
-    'SOCIAL_AUTH_FACEBOOK_KEY': '2008148216081463',
-    'SOCIAL_AUTH_FACEBOOK_SECRET': '75f855457ff18db012e57ccaaf3470eb',
-    'FB_TEST_EMAIL': 'open_ejvbpxf_user@tfbnw.net',
-    'FB_TEST_PASS': 'asdfasdfdsfdsfadfdsflsdflglfg',
-    'SOCIAL_AUTH_AZURE_TENANT_KEY': None,
-    'SOCIAL_AUTH_AZURE_TENANT_SECRET': None,
-    'SOCIAL_AUTH_AZURE_TENANT_ID': None,
-    'STUDENT_SIS_AUTH_SECRET': 'TEST_KEY',
+    # Credentials for a test application for Semester.ly (+ Google/Facebook)
```

Comment:
We should make a note/link to the docs so they know they'll have to update some of these credentials.

Reply:
This is already in the docs